tasks/add_new_prebid_partner.py

```diff
@@ -301,10 +339,42 @@ def main():
     len(placements)
   )
 
+  # In the case where no inventory is being used, make sure at least one 
+  # creative is created
+  if not num_creatives > 0:
+    num_creatives = 1
```

Comment:
Let's actually raise an exception (earlier in the settings validation where we first call `getattr`) and force the user to set a number in this case. That's better than having their ads not show because they have too few creatives (see https://github.com/kmjennison/dfp-prebid-setup/issues/13#issuecomment-288207392).